src/azul/indexer/document.py

```diff
@@ -365,8 +411,12 @@ def from_index(self, value: str) -> Optional[str]:
         else:
             return value
 
+    @property
```

Comment:
I was hoping these would become redundant. You thumbed up my previous comment about this, but didn't actually remove the method. So either I am wrong, in which case there should be a short comment explaining the deviation, or I am right and you forgot in which case you shouldn't have thumbed up the comment. What am I missing?

Reply:
For PL:

https://github.com/DataBiosphere/azul/pull/3325#discussion_r745171506

the suggested default method does not account for self.native_type being a union of some type and None, because of this we have to have multiple overrides.

Handle the logic in FieldType, or add a new subclass (NullableFieldType) both can get rid of the self.native_type_ shadow.